src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,89 @@
+import { useParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { PersonLink } from '../PersonLink/PersonLink';
+import { Person } from '../../types';
+
+export type PropsPeopleTable = {
+  people: Person[],
+};
+
+export const PeopleTable: React.FC<PropsPeopleTable> = ({ people }) => {
+  const { slug } = useParams();
+
+  const findParentOfPerson = (parentName: string | null) => {
```

Comment:
make sense to use useMemo here, because it's essential to performance

Reply:
done